lightning/src/ln/channelmanager.rs

```diff
@@ -169,6 +169,25 @@ pub enum PendingHTLCRouting {
 		/// The absolute CLTV of the inbound HTLC
 		incoming_cltv_expiry: Option<u32>,
 	},
+
+	/// An HTLC which should be forwarded on to another Trampoline node.
+	TrampolineForward {
+		/// The onion shared secret we build with the sender (or the preceding Trampoline node) used
+		/// to decrypt the onion.
+		///
+		/// This is later used to encrypt failure packets in the event that the HTLC is failed.
+		incoming_shared_secret: [u8; 32],
+		/// The onion which should be included in the forwarded HTLC, telling the next hop what to
+		/// do with the HTLC.
+		onion_packet: msgs::TrampolineOnionPacket,
+		/// The node ID of the Trampoline node which we need to route this HTLC to.
+		node_id: NodeId, // This should be NonZero<u64> eventually when we bump MSRV
+		/// Set if this HTLC is being forwarded within a blinded path.
+		blinded: Option<BlindedForward>,
+		/// The absolute CLTV of the inbound HTLC
+		incoming_cltv_expiry: Option<u32>,
+	},
+
```

Comment:
nit: should skip the newline here and above for consistency with the other variants